pkg/apis/apps/register.go

```diff
@@ -50,6 +50,7 @@ func addKnownTypes(scheme *runtime.Scheme) {
 		&PetSet{},
 		&PetSetList{},
 		&api.ListOptions{},
+		&api.DeleteOptions{},
```

Comment:
@smarterclayton @lavalamp remind me why we can't reference the one actual API version we have that defines these?